lighthouse-core/test/fixtures/lantern-baseline-computed-values.json

```diff
@@ -1,81 +1,81 @@
+    {"url": "https://www.instagram.com/", "roughEstimateOfFCP": 1056, "optimisticFCP": 1056, "pessimisticFCP": 1056, "roughEstimateOfFMP": 4043, "optimisticFMP": 3968, "pessimisticFMP": 4118, "roughEstimateOfTTI": 6345, "optimisticTTI": 5440, "pessimisticTTI": 7251, "roughEstimateOfSI": 3111, "optimisticSI": 822, "pessimisticSI": 3401, "roughEstimateOfLCP": 5631, "optimisticLCP": 5556, "pessimisticLCP": 5706},
+    {"url": "https://www.irs.gov/", "roughEstimateOfFCP": 2182, "optimisticFCP": 2182, "pessimisticFCP": 2182, "roughEstimateOfFMP": 3469, "optimisticFMP": 2948, "pessimisticFMP": 3990, "roughEstimateOfTTI": 15941, "optimisticTTI": 14474, "pessimisticTTI": 17407, "roughEstimateOfSI": 5174, "optimisticSI": 818, "pessimisticSI": 6582, "roughEstimateOfLCP": 4618, "optimisticLCP": 4044, "pessimisticLCP": 5192},
+    {"url": "https://www.java.com/en/", "roughEstimateOfFCP": 956, "optimisticFCP": 956, "pessimisticFCP": 956, "roughEstimateOfFMP": 956, "optimisticFMP": 956, "pessimisticFMP": 956, "roughEstimateOfTTI": 3453, "optimisticTTI": 3097, "pessimisticTTI": 3810, "roughEstimateOfSI": 1203, "optimisticSI": 484, "pessimisticSI": 1194, "roughEstimateOfLCP": 3196, "optimisticLCP": 3196, "pessimisticLCP": 3196},
+    {"url": "https://www.linkedin.com/", "roughEstimateOfFCP": 1903, "optimisticFCP": 1903, "pessimisticFCP": 1903, "roughEstimateOfFMP": 1903, "optimisticFMP": 1903, "pessimisticFMP": 1903, "roughEstimateOfTTI": 11440, "optimisticTTI": 9843, "pessimisticTTI": 13037, "roughEstimateOfSI": 2073, "optimisticSI": 655, "pessimisticSI": 2164, "roughEstimateOfLCP": 2196, "optimisticLCP": 2155, "pessimisticLCP": 2237},
+    {"url": "https://www.metacafe.com/", "roughEstimateOfFCP": 1900, "optimisticFCP": 1900, "pessimisticFCP": 1900, "roughEstimateOfFMP": 1900, "optimisticFMP": 1900, "pessimisticFMP": 1900, "roughEstimateOfTTI": 6467, "optimisticTTI": 5990, "pessimisticTTI": 6943, "roughEstimateOfSI": 3513, "optimisticSI": 769, "pessimisticSI": 4134, "roughEstimateOfLCP": 5198, "optimisticLCP": 5198, "pessimisticLCP": 5198},
+    {"url": "https://www.mgid.com/ru", "roughEstimateOfFCP": 2164, "optimisticFCP": 2164, "pessimisticFCP": 2164, "roughEstimateOfFMP": 4583, "optimisticFMP": 4184, "pessimisticFMP": 4983, "roughEstimateOfTTI": 16150, "optimisticTTI": 15036, "pessimisticTTI": 17265, "roughEstimateOfSI": 6699, "optimisticSI": 983, "pessimisticSI": 8575, "roughEstimateOfLCP": 8899, "optimisticLCP": 8500, "pessimisticLCP": 9299},
+    {"url": "https://www.mlb.com/", "roughEstimateOfFCP": 3186, "optimisticFCP": 3186, "pessimisticFCP": 3186, "roughEstimateOfFMP": 3186, "optimisticFMP": 3186, "pessimisticFMP": 3186, "roughEstimateOfTTI": 40427, "optimisticTTI": 32360, "pessimisticTTI": 48494, "roughEstimateOfSI": 19986, "optimisticSI": 1316, "pessimisticSI": 28299, "roughEstimateOfLCP": 4634, "optimisticLCP": 4312, "pessimisticLCP": 4956},
```

Comment:
Callout for update LCP values on this site

Reply:
Seems like a good test case for this since it's now `roughEstimateOfLCP: 4634` while the "real" LCP was 9399.

Is the cause of the change an unfinished request and does it look reasonable to exclude it?

Reply:
Yes. This difference is from negative `endTime` requests.

This change doesn't make much difference on a new run of https://www.mlb.com.

I did notice a large drop in LCP time for a new run of https://mail.ru/ (11s -> 4s), but the LCP with DT throttling is ~2.4s so this seems reasonable.